frontend/src/pages/profile/components/user-activity/user-activity.tsx

```diff
@@ -13,18 +18,27 @@ type Properties = {
 
 const UserActivity: React.FC<Properties> = ({ userActivity }) => {
   const uniqueMonths: string[] = getUniqueMonths(userActivity);
+  const dateFormatter = new Intl.DateTimeFormat(DATE_LOCALE, DATE_OPTIONS);
 
-  const activityStatistic = userActivity.map((activity) => {
+  const activityStatistic = userActivity.map((activity, index) => {
     const { count, date } = activity;
     const activityBreakpoint = getActivityBreakpoint(count);
+    const localDate = dateFormatter.format(new Date(date));
```

Comment:
Let's use getFormattedDate helper, as it's already added

Reply:
Is there function to convert date to the current timezone?
Because from server dates come one day shifted

Reply:
I tried to find such a method there but it seems like it hasn't a method to automatically convert a date to the current timezone
This is necessary because the dates from the server are shifted by one day

Reply:
`format` method in date-fns has separate `options` param - couldn't utc cases be resolved by it?

Reply:
Oh, yes, we could, at first I just didn't want to use enUs as the locale and wanted the locale to be defined automatically but I changed my mind when I saw Ukrainian words in the date )